src/components/widgets/filesystem/FileSystem.vue

```diff
@@ -621,11 +735,31 @@ export default class FileSystem extends Mixins(StateMixin, FilesMixin, ServicesM
     SocketActions.serverFilesMove(src, dest)
   }
 
-  handleRemove (file: AppFile | AppFileWithMeta | AppDirectory | (AppFile | AppFileWithMeta | AppDirectory)[]) {
+  handleRemove (file: FileBrowserEntry | AppFileWithMeta | (FileBrowserEntry | AppFileWithMeta)[], callback?: () => void) {
```

Comment:
Is `callback` here actually in use? I can't seem to find it anywhere...

Reply:
It's used [here](https://github.com/fluidd-core/fluidd/pull/610/files#diff-7a8df2f9b1d14dee2891d7abf461ac805319ba4cff8599196b6b7e8aba91911cR51)